pkg/daemon/ceph/osd/volume.go

```diff
@@ -604,6 +606,12 @@ func getCephVolumeLVMOSDs(context *clusterd.Context, clusterName string, cephfsi
 			if osd.Type == "journal" {
 				store = "filestore"
```

Comment:
this will never be the case anymore, right?

Reply:
Right, I think it's still there for backward compatibility but I think we could remove it. Perhaps I was over-conservative.